[ZT] Secure your Internet traffic and SaaS apps (#12460)
cloudflare/cloudflare-docs: content/learning-paths/_partials/zero-trust/_create-cloudflare-account.md

---
_build:
  publishResources: false
  render: never
  list: never
---

To create a new Cloudflare account:

1. [Sign up](https://dash.cloudflare.com/sign-up) on the Cloudflare dashboard.

2. To secure your account, enable [two-factor authentication](/fundamentals/setup/account/account-security/2fa/).

3. If you have a Cloudflare contact (Enterprise only), ask them to set up your account as a multi-user organization. Account members will need:

>     - [**Access** permissions](/cloudflare-one/roles-permissions/) to view or edit applications for your private network.

Reviewer: ```suggestion
    - [**Access** permissions](/cloudflare-one/roles-permissions/) to view or edit applications and Access policies.
```

>     - [**Access** permissions](/cloudflare-one/roles-permissions/) to view or edit applications for your private network.

Reviewer: Suggest making this more generic so that it applies to all learning paths.

>     - [**Gateway** permissions](/cloudflare-one/roles-permissions/) to view or edit policies for your applications.

Reviewer: ```suggestion
    - [**Gateway** permissions](/cloudflare-one/roles-permissions/) to read or edit Gateway policies.
```

>     - [**PII** permissions](/cloudflare-one/roles-permissions/#cloudflare-zero-trust-pii) to view user information in network activity logs.

Reviewer: ```suggestion
    - [**PII** permissions](/cloudflare-one/roles-permissions/#cloudflare-zero-trust-pii) to view user information in Gateway activity logs.
```
